README.md

```diff
@@ -28,3 +28,7 @@ We are aiming to make Chart.js the only popular JavaScript library that is both
 npm install
 gulp
 ```
+
+## Examples
+
+Out of the box examples are available here : https://loicbourgois.github.io/chartjs-chart-financial/
```

Comment:
we should probably change loicbourgois to chartjs

Reply:
done
But keep in mind that this will return a 404 until GitHub Pages is properly configured for this repo.

Reply:
ok. I'll set it up after merging this PR